docset/windows/dism/add-windowscapability.md

```diff
@@ -43,25 +43,32 @@ The **Add-WindowsCapability** cmdlet acquires a Windows capability package from
 
 ## EXAMPLES
 
-### Example 1: Add a Windows capability package to an image
+### Example 1: Add a Windows capability package to the running OS via the Windows Update client
 ```
-PS C:\> Add-WindowsCapability -Name "Language.TextToSpeech~~~fr-FR~0.0.1.0" -Path "C:\offline" -Source "C:\Windows\winsxs"
+PS C:\> Add-WindowsCapability -Online -Name "Msix.PackagingTool.Driver~~~~0.0.1.0"
 ```
+This command adds a Windows capability package to the running operating system. Because no source is specified, the Windows Update client will download the necessary package. It requires either an active Internet connection or an active network connection to the local Windows Server Update Services (WSUS) server.
 
-This command adds a Windows capability package to the operating system image specified by the *Name* parameter at the path C:\offline.
-The *Source* parameter specifies the location of required files.
+### Example 2: Add a Windows capability package to the running OS using a locally stored package file
+```
+PS C:\> Add-WindowsCapability -Online -Name "Msix.PackagingTool.Driver~~~~0.0.1.0" -Source "E:\" -LimitAccess
+```
+
+This command adds a Windows capability package specified by the *Name* parameter, to the running operating system. The *Source* parameter specifies the location of required files. For instance, if the running OS is a copy of Windows 10 version 1809, the `Msix-PackagingTool-Driver-Package~31bf3856ad364e35~amd64~~.cab` file must present at `E:\`.
+
+If the package specified by the *Name* parameter is already installed, this command does not return an error message, regardless of whether the required files are present at `E:\`.
 
-### Example 2: Add a Windows capability package to the local host
+### Example 3: Add a Windows capability package to an image
 ```
-PS C:\> Add-WindowsCapability -Name "Language.TextToSpeech~~~fr-FR~0.0.1.0" -Online -LimitAccess -Source "C:\Windows\winsxs"
+PS C:\> Add-WindowsCapability -Path "C:\mount\Windows" -Name "Msix.PackagingTool.Driver~~~~0.0.1.0" -Source "E:\"
 ```
 
-This command adds a windows capabilities package the local host while the operating system continues to run.
+This command adds a Windows capability package specified by the *Name* parameter,  to the operating system image at the path C:\mount\Windows. The *Source* parameter specifies the location of required files. For instance, if the mount point contains a copy of Windows 10 version 1809, the `Msix-PackagingTool-Driver-Package~31bf3856ad364e35~amd64~~.cab` file must present at `E:\`.
 
 ## PARAMETERS
 
 ### -LimitAccess
-Indicates that this cmdlet does not access Windows Update for the source package.
+Indicates that this cmdlet does not access Windows Update for the source package. Only applies when `-Online` switch is specified; otherwise, ignored.
```

Comment:
```suggestion
Indicates that this cmdlet does not access Windows Update for the source package. Only applies when `-Online` switch is specified; otherwise, it will be ignored.
```

Reply:
Hi, Andres. 😊 This seems like a deviation from the document's style. In this documents, when the subject of the sentence is "this switch", the subject is omitted entirely and the sentence starts with a verb. Example:

- Indicates that this cmdlet does not ...
- Specifies the maximum output level ...
- Specifies the full path ...
- Specifies the identity of the capability ...
- Indicates that the cmdlet operates on ...
- Specifies the full path to the root directory ...
- Specifies a temporary directory that ...
- Specifies the location of the files that ...
- Specifies the path to the location ...
- Specifies the path to the Windows directory ...

So, why did you suddenly feel the need to add the subject, and with a pronoun no less? And why the future tense?

Reply:
ok it seems you handle this topic much better than me, leave it that way.

Reply:
My pleasure. 😊

Reply:
I suggest "Unless `-Online` switch is specified, is ignored," or if you insist  "otherwise ignored" (without the comma).

Reply:
How is this:

> Indicates that this cmdlet does not query Windows Update for source packages when servicing a live OS. Only applies when the `-Online` switch is specified.